alerts.jsonnet

```diff
@@ -1 +0,0 @@
-std.manifestYamlDoc((import 'mixin.libsonnet').prometheusAlerts)
```

Comment:
why are we deleting this ? i like it it lets you render to yaml locally

Reply:
you can already do this

```
jsonnet -J vendor prometheus-rules.jsonnet
```